vendor/CMakeLists.mkbootimg.txt

```diff
@@ -1,3 +1,9 @@
-install(PROGRAMS mkbootimg/mkbootimg.py DESTINATION bin RENAME mkbootimg)
+set(MKBOOTIMG_SCRIPTS_DIR "${CMAKE_INSTALL_FULL_DATADIR}/android-tools/mkbootimg")
+install(PROGRAMS mkbootimg/mkbootimg.py DESTINATION ${MKBOOTIMG_SCRIPTS_DIR})
+add_custom_target(mkbootimg_symlink ALL COMMAND ${CMAKE_COMMAND} -E create_symlink
```

Comment:
I see that this target is called even if I do not modify any sources

```
ninja: Entering directory `build'
[1/1] cd /home/anatol/sources/android-tools/build/vendor && /usr/bin/cmake -E create_symlink /usr/share/android-tools/mkbootimg/mkbootimg.py /home/anatol/sources/android-tools/build/vendor/mkbootimg
```

what is this custom target used for? Do we need it here?

Reply:
Nothing, this was one of the ways I found to create the final installed `/usr/bin/mkbootimg` symlink file itself (which the `install()` below places in `/usr/bin/`); I already acknowledged it wasn't so nice in the description of the PR.

Please do replace it with something else if you know a better way!

Reply:
I am not really an CMake expert cc @nmeum for more help

@nmeum Do you know how to get `install()` target to avoid re-run here in case of a no-op build?

Reply:
`add_custom_target` is not intended to have any output and is thus always run. You could maybe try to work around that via [`add_custom_command`](https://cmake.org/cmake/help/latest/command/add_custom_command.html)? I haven't looked into using that to create symlinks as part of the installation though.